src/BoundPattern.jl

```diff
@@ -1,4 +1,6 @@
+using Base: ImmutableDict
 
+# Unfortunately, using a type alias instead of the written-out type tickles a Julia bug.
```

Comment:
has an issue been filed for this in the Julia repo? If so, could you link to it here?